.config.sample.env

```diff
@@ -19,12 +19,9 @@ export BOOTSTRAP_METALLB_LB_RANGE=""
 # The load balancer IP for traefik, choose from one of the available IPs above
 # e.g. 192.168.1.220
 export BOOTSTRAP_METALLB_TRAEFIK_ADDR=""
-# Your personal pgp key
-# e.g. 772154FFF783DE317KLCA0EC77149AC618D75581
-export BOOTSTRAP_PERSONAL_KEY_FP=""
-# The flux pgp key
-# e.g. AB675CE4CC64251G3S9AE1DAA88ARRTY2C009E2D
-export BOOTSTRAP_FLUX_KEY_FP=""
+# Age Public Key - string should start with age
```

Comment:
Do you want to check for this prefix in the verification step? To avoid that folks drop in their GPG pubkey fp and things don't work for them.

Reply:
Good idea, added in latest commit